nexus/auth/src/authz/oso_generic.rs

```diff
@@ -172,8 +172,7 @@ pub fn make_omicron_oso(log: &slog::Logger) -> Result<OsoInit, anyhow::Error> {
 ///
 /// There's currently just one enum of Actions for all of Omicron.  We expect
 /// most objects to support mostly the same set of actions.
-#[derive(Clone, Copy, Debug, Eq, PartialEq)]
-#[cfg_attr(test, derive(strum::EnumIter))]
```

Comment:
Another example of "I need this trait in `policy_test`, but it's not part of the same crate anymore"

Reply:
We could, potentially, make the `strum` dependency optional and make this:
```rust
#[cfg_attr(feature = "strum", derive(strum::EnumIter)]
```
and then have `policy_test` enable the `strum` feature in its dev-dependency on `authz`...but i'm not sure if that's *really* worth the effort, up to you.

Reply:
Yeah, I have mixed feelings on this, because that feels "logically correct", but it's also a pain-in-the-butt to need to test an inflated matrix of possible features.

My belief in *this* case is that the value of (feature flag + reduced API) is smaller than the value of (no feature, slightly larger API). But that's totally an opinion. Would be willing to update that if you disagree!